en/getting_started/sensor_selection.md

```diff
@@ -1,61 +1,60 @@
+
+The minimal set of sensors is incorporated into [Pixhawk Series](../flight_controller/pixhawk_series.md) flight controllers (and may also be in other controller platforms). Additional/external sensors can be attached to the controller.
+
+Below we describe some of the sensors. At the end there are links to information about [sensor wiring](#wiring).
+
 
 ## GPS
 
-There are many types of GPS you can use with Pixhawk and PX4. We
-recommend using GPS + Compass solution like the original 3DR GPS or
-alternative LEA 6H, NEO 7M, NEO 8 GPS+Compass solutions.
+Many types of GPS can be used with PX4. When using Pixhawk-series flight controllers we recommend using a *combined GPS + Compass*.
+
+> **Tip** The Pixhawk internal compass is close to other electronics, and may be susceptible to electromagnetic noise. An external compass/GPS mounted on a pedestal or wing (for fixed-wing) away from the rest of the system is recommended.
+
+Options include the original 3DR GPS, LEA 6H, NEO 7M, NEO 8 GPS+Compass solutions.
 
 ![3DR GPS_Top](../../images/gps_3dr_top_1.jpg) 
 ![GPS + Compass](../../images/gps_compass.jpg)
 
-Because the internal compass of Pixhawk is close to other electronics it
-could be susceptible to electromagnetic noise. So when we use the
-external Compass we can locate it away from electronics on a pedestal or
-at the wing of the plane.
 
 ## Airspeed
 
-The Airspeed sensor is mandatory for most of the Airplane Frames and for
-all VTOL Frames. It is very important sensor because the PX4 autopilot
-does not have other means to detect stall, except with the airspeed
-sensor. For the airplane it is the airspeed that guarantees lift not
-ground speed. We recommend using the digital airspeed sensor.
+The Airspeed sensor is required for most of fixed-wing frames and for all VTOL frames.
+
+This sensor is so important because the autopilot does not have other means to detect stall. 
+For fixed-wing vehicles it is the airspeed that guarantees lift not ground speed!
+
+We recommend using a digital airspeed sensor.
 
 ![Digital airspeed sensor](../../images/digital_airspeed_sensor.jpg)
 
 ## Distance
 
-There are more and more affordable Lidar Rangefinder options. These
-sensors are used during the final phase of landing for precise altitude
-measuring. They are also used for avoidance. PulsedLight launched their
-Lidar-Lite2 Laser Rangefinder. as an economical laser based measurement
-solution supporting a wide variety of applications like in drones. It
-measures distance, velocity and signal strength of cooperative and non
-cooperative targets at distances from zero to more than 40 meters.
-Offering the highest performance available in a single beam ranging
-sensor in its class.
+Distance sensors are used for precision landing, object avoidance and terrain following.

```

Comment:
Updated to remove specific reference to Lidar-lite (or any other product). What I wanted to do here is explain what they are for, and provide links to more information. Think it works better!

Anything else we need to say for the introductory audience?

Reply:
yep, that's better